### PR TITLE
more documentation and feedback from qa

### PR DESCRIPTION
**Pull request by StigNorland**

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -188,86 +229,125 @@ paths:
               schema:
                 $ref: "#/components/schemas/Problem"
 
-  /resources:
+  /resources2:
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:232]: Can this be omitted? Do we still serve `/resources2?`

File: docs/openapi.yaml [hunk 2]
```diff
@@ -552,62 +668,132 @@ paths:
...
-    categoryParam:
-      name: category
+    abstractParam:
+      name: abstract
       in: query
-      description: "Returns documents with this category code, only one category code is allowed."
-      example: "JournalArticle"
+      description: "Returns document with these words in the abstract."
       required: false
+      examples:
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:676]: Nice

File: docs/openapi.yaml [hunk 3]
```diff
@@ -552,62 +668,132 @@ paths:
...
+      description: "Returns document with similar Author names."
+      example: "John Doe"
       required: false
       schema:
         type: string
     contributorParam:
       name: contributor
       in: query
-      description: "returns document with Author's name or person id(URI)."
+      description: "Returns document with Author's id(URI)."
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:772]: Is a contributor an author? I have no idea, am just curious.

Comment (StigNorland) [reply docs/openapi.yaml:621]: yeah, kind of (co-author).

File: docs/openapi.yaml [hunk 4]
```diff
@@ -698,203 +884,258 @@ components:
...
+      example: "978-3-16-148410-0"
       required: false
       schema:
         type: string
     isbnNotParam:
       name: isbn_not
       in: query
-      description: "returns document without these ISBN ids, seperated by comma (+)."
+      description: "Returns document without these ISBN ids."
+      example: "978-3-16-148410-0"
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:924]: Give an example of isbn without dashes? I think we support that?

File: docs/openapi.yaml [hunk 5]
```diff
@@ -698,203 +884,258 @@ components:
...
       schema:
-        type: string
-    orcidNotParam:
-      name: orcid_not
+        $ref: "#/components/schemas/LicenseValues"
+    licenseParam:
+      name: license
       in: query
-      description: "returns document without these ORCID ids, seperated by comma (+)."
+      description: "Returns document with this license key (without version)."
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:971]: "Returns document`s` with..." (plural?)

File: docs/openapi.yaml (hunk 4 above)
Comment (StigNorland) [reply docs/openapi.yaml:924]: Actually, the search api needs to be able to wash incomming search phrases, for formatting, and in some cases add formatting

File: docs/openapi.yaml [hunk 6]
```diff
@@ -1561,14 +2018,40 @@ components:
...
+              DegreeMaster,DegreePhd,Encyclopedia,EncyclopediaChapter,ExhibitionCatalog,ExhibitionCatalogChapter,ExhibitionProduction,FeatureArticle,Introduction,JournalArticle,JournalCorrigendum,
+              JournalInterview,JournalIssue,JournalLeader,JournalLetter,JournalReview,Lecture,LiteraryArts,MediaBlogPost,MediaFeatureArticle,MediaInterview,MediaParticipationInRadioOrTv,
+              MediaReaderOpinion,MovingPicture,MusicPerformance,NonFictionChapter,NonFictionMonograph,OtherPresentation,OtherStudentWork,PerformingArts,PopularScienceArticle,PopularScienceChapter,
+              PopularScienceMonograph,ProfessionalArticle,ReportBasic,ReportBookOfAbstract,ReportPolicy,ReportResearch,ReportWorkingPaper,StudyProtocol,Textbook,TextbookChapter,VisualArts ]
+    ContextValues:
+      type: string
+      enum: [ Anthology,Artistic,Book,Degree,Event,ExhibitionContent,Journal,MediaContribution,MediaContributionPeriodical,Report,UnconfirmedJournal,UnconfirmedMediaContributionPeriodical ]
+    LicenseValues:
+      type: string
+      enum: [ 'CC BY', 'CC BY-NC', 'CC BY-NC-ND', 'CC BY-NC-SA', 'CC BY-ND', 'CC BY-SA', 'CC0' ]
```
Comment (joachimjorgensen) [inline docs/openapi.yaml:1903]: We do have a license "rightsReserved" (http://rightsstatements.org/vocab/InC/1.0/). I that one searchable?

**Review by joachimjorgensen — APPROVED** [review]
Very nice! Feels good to have up-to-date documentation 😄

File: docs/openapi.yaml (hunk 6 above)
Comment (StigNorland) [reply docs/openapi.yaml:1903]: Not sure, this is the list of names that Terje gave me.

**Pull request merged**